src/layouts/Folders.jsx

```diff
@@ -91,9 +90,10 @@ const Folders = ({ match, location }) => {
     // parse contents of current folder directory
     useEffect(() => {
       if (folderContents && folderContents.data) {
-        const parsedFolderContents = parseDirectoryFile(folderContents.data.content)
+        const { order: parsedFolderContents, output: parsedFolderOutput } = parseDirectoryFile(folderContents.data.content)
         setDirectoryFileSha(folderContents.data.sha)
         setParsedFolderContents(parsedFolderContents)
+        setParsedFolderOutput(parsedFolderOutput)
```

Comment:
Nit: could we rename the variables retrieved so that it doesn't have the same name as the state variable?